src/commcare_cloud/environment/main.py

```diff
@@ -324,6 +387,13 @@ def get_hostname(self, sshable_hostname, full=True):
         return hostname
 
 
+def setup_environment(env_name, input_argv):
```

Comment:
Maybe it'd be better to call this `make_argv_available_to_environment`. A name like that more clearly signals that this has a special, possibly hacky, purpose and that it's not a good place to highjack for your own use. In this case, `input_argv` is fundamentally an optional piece of info, used only for this ancillary feature; it'd probably be a bad idea to hijack this setup function to inject fundamentally necessary properties. A more what-this-does name would signal to someone who wanted to repurpose this entry point that it'd be up to them to make it a more general system, and rename accordingly, if they did want to go that route.

Reply:
https://github.com/dimagi/commcare-cloud/pull/2733/commits/abdf3890ae99b9c5358edec015a3f4e76f94eac8 - decided to use `sys.argv` directly rather than passing `input_arv` around.